docs/en/release-process.md

```diff
@@ -84,8 +88,8 @@ Check if all plugin dependencies are proper versions and not SNAPSHOT!
 ## Build, commit and push
 
 ```
-rm -rf node_modules yarn.lock && mvn clean install \
-&& git add yarn.lock pom.xml package.json \
+rm -rf node_modules && ./gradlew build \
```

Comment:
I think we don't need the removal of `node_modules` any more

Reply:
```suggestion
./gradlew build \
```